apps/common/src/main/resources/db/migration/canton-network/postgres/stable/V040__active_parties.sql

```diff
@@ -17,8 +17,10 @@ begin
         insert into active_parties (store_id, party, closed_round)
         select store_id, party, max(closed_round)
         from round_party_totals
-        group by store_id, party;
+        group by store_id, party
```

Comment:
is it safe to modify an old migration? I thought flyway blows up if that migration has already been applied?